base/show.jl

```diff
@@ -61,6 +61,11 @@ The following properties are in common use:
  - `:displaysize`: A `Tuple{Int,Int}` giving the size in rows and columns to use for text
    output. This can be used to override the display size for called functions, but to
    get the size of the screen use the `displaysize` function.
+ - `:typeinfo`: a type specification characterizing the information already expressed
+   concerning the type of the object about to be displayed. This is mainly useful when
+   displaying a collection of object of the same type, so that redundant type information
```

Comment:
"objects". Close the parentheses.

Maybe a simpler example would be easier to understand, e.g. with `[Float16(1)]`? The text should say what's the value of the argument in that case.